raster/src/main/scala/geotrellis/raster/render/ColorMap.scala

```diff
@@ -143,6 +143,15 @@ object ColorMap {
     }.toOption
   }
 
+  /** Retrieve a "breaks string" from [[ColorMap]] data.
+    * The opposite of the [[fromString]] methods.
+    */
+  def breaksString[T <: AnyVal](breaksToColors: Map[T, Int]): String = {
+    breaksToColors
+      .toStream
+      .map({ case (k,v) => s"${k}:${Integer.toHexString(v)}"})
+      .mkString(";")
+  }
 }
```

Comment:
I don't want to have to write two functions just to distinguish between the `Int` and `Double` keys in the map, here. The function's implementation would be exactly the same.

What I really want to write is:

```scala
def breakString[T <: Num](...): String = { ... }
```
but Scala doesn't have a specialized number typeclass.